lib/index.js

```diff
@@ -301,7 +301,12 @@ HttpCache.prototype = {
     if (options.json) {
       /* Use the cached response */
       try {
-        var parsed = JSON.parse(cachedContent.body);
+        var parsed;
+        if( typeof cachedContent.body === 'object' ) {
+            parsed = cachedContent.body;
```

Comment:
This needs to return a deep clone, otherwise client changes to the object would bleed between requests.